### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by mrblonde91**
Readme plus put cluster id back in

### CODE REVIEW

File: 8/readme.md [hunk 1]
```diff
@@ -1,41 +1,63 @@
...
+![Container ports info](imgs/1.png)
 
-### silos
 
-As in example 7, following scripts can be used to run silos:
+* Internally the ports are defaulted and externally an available one is selected.
+![Dynamo table info](imgs/2.png)
+* A sample of the dynamo data is displayed above, the cluster id acts as a unique identifier for each cluster. So multiple clusters can be stored 
+in the same table if the user wishes. Clusters can also contain different versions of silos eg for testing purposes.
+### Running Silo local -> Aws Cluster
```
Comment (PiotrJustyna) [inline 8/readme.md:29]: I think this could be made clearer: `running local and ECS-hosted silos in one cluster`? Or something better phrased.

**Review by PiotrJustyna — COMMENTED** [review]
LGTM, just minor comments. Can you also update the main readme please?

File: 8/ops/Convenience/run-silo-1-docker.sh [hunk 2]
```diff
@@ -12,6 +12,6 @@ AWSREGION="us-west-2"
 ISLOCAL=true
 
 docker build -t silo-host-cluster -f ./ops/SiloHost/Dockerfile ./ &&
-  docker run -d -e AWS_REGION -e AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN -e AWS_ACCESS_KEY_ID\
-  -e ADVERTISEDIP=$ADVERTISEDIP  -e GATEWAYPORT=$GATEWAYPORT -e SILOPORT=$SILOPORT -e DASHBOARDPORT=$DASHBOARDPORT\
+  docker run -it -e AWS_REGION -e AWS_SECRET_ACCESS_KEY -e AWS_SESSION_TOKEN -e AWS_ACCESS_KEY_ID\
```
[the remark this answers is not in the document]
Comment (mrblonde91) [reply 8/ops/Convenience/run-silo-1-docker.sh:15]: Ah, was testing...

File: 8/readme.md [hunk 3]
```diff
@@ -1,41 +1,63 @@
...
+* Run the template to pull the image and it will start creating the containers.
+
+![Container ports info](imgs/1.png)
 
-### silos
 
-As in example 7, following scripts can be used to run silos:
+* Internally the ports are defaulted and externally an available one is selected.
+![Dynamo table info](imgs/2.png)
+* A sample of the dynamo data is displayed above, the cluster id acts as a unique identifier for each cluster. So multiple clusters can be stored
```
[the remark this answers is not in the document]
Comment (mrblonde91) [reply 8/readme.md]: Done

File: 8/readme.md (hunk 1 above)
Comment (mrblonde91) [reply 8/readme.md]: Done

**Review by PiotrJustyna — COMMENTED** [review]
LGTM, thanks. What about the main readme?

**Review by PiotrJustyna — APPROVED** [review]
LGTM, thank you!

**Pull request merged**